src/minimega/stats.go

```diff
@@ -108,6 +114,24 @@ func cliHost(c *minicli.Command, resp *minicli.Response) error {
 	return nil
 }
 
+func hostStatsUptime() (string, error) {
```

Comment:
Hah, uptime before was how long minimega has been running. Are both useful? I was thinking we were going to kill the `uptime` API and report how long minimega has been running in `host uptime` but maybe that's confusing.

@djfritz